### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by maksimandrianov**
так как ждем ответа по pl у @strebski, строки пока не генерировал

### CODE REVIEW

**Comment by maksimandrianov** [comment]
@strebski please take a look

**Comment by tatiana-yan** [comment]
generate_localizations?

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -1975,7 +1975,37 @@
...
+    pt-BR = Estacionamento
+    ro = Loc de parcare
+    es = Plaza de aparcamiento
+    es-MX = Plaza de aparcamiento
+    sv = Parkeringsplats
+    th = พื้นที่จอดรถ
+    tr = Park alanı
+    uk = Паркувальне місце
+    vi = Chỗ để xe
+    el = Χώρος στάθμευσης
```
Comment (mpimenov) [inline data/strings/types_strings.txt]: el, sk, fa after zh-Hant

**Comment by mpimenov** [comment]
@strebski PTAL

**Comment by strebski** [comment]
Pardon, I was outside Github last data:
[strings] type.landuse.churchyard
pl = Teren przykościelny

[strings] type.amenity.parking_space
pl = Miejsce parkingowe

Cannot update strings.txt myself, working only on smartphone - or give me pls 2 days :-)

**Comment by mpimenov** [comment]
@strebski The translation by our contractor is in conflict with yours. Which do you think is better?

**Comment by strebski** [comment]
"Miejsce parkingowe" is indisputable. Simply correct.
We can think about "Teren przykościelny" - alternative is "Teren kościelny"; but first (my proposition) is strictly related with place (church and it's surroundings) when second option is related (in Polish language) more with property (as a terrain which belongs to the church).
So I suggest my both propositions as correct in our map context.

**Comment by mpimenov** [comment]
The kościelny/przykościelny look similar enough to me, I think we'll leave the "przykościelny" which according to you has the meaning "church and it's surroundings".

I'm more interested in the distinction between "Dziedziniec" and "Teren".

**Comment by mpimenov** [comment]
@strebski We'll go with your original suggestion for now. Thanks.

**Comment by strebski** [comment]
"Dziedziniec" is very old-fashioned and elegant word, more related with castles and manors.
"Teren" is neutral and most popular (simple) word.
All best!

**Pull request merged**